From 3b90d24fd478ae3804178f6f70677d260ffbc8b7 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Thu, 12 Jul 2018 14:17:24 +0800
Subject: [PATCH 195/205] ARM: dts: am335x-evmsk: Enable PRU-ICSS and the
 child PRU nodes

The PRU-ICSS processor bus node, subsystem node and their associated
child PRU remote processor nodes were left in disabled state in the
base am33xx.dtsi file. PRU-ICSS is supported on the AM335x SK EVM
board, and so all these PRU-ICSS nodes are enabled.

The corresponding nodes can be disabled later on if there is no
use-case defined to use a remote PRU core or the PRU-ICSS subsystem
if both its PRU cores are unused.
commit 2c9d7b979bd9b1138d968ef76aae06df3aa0d316

Signed-off-by: Suman Anna <s-anna@ti.com>
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am335x-evmsk.dts |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-evmsk.dts b/arch/arm/boot/dts/am335x-evmsk.dts
index a4067af..b153440 100644
--- a/arch/arm/boot/dts/am335x-evmsk.dts
+++ b/arch/arm/boot/dts/am335x-evmsk.dts
@@ -730,3 +730,19 @@
 	clocks = <&clk_32768_ck>, <&clkdiv32k_ick>;
 	clock-names = "ext-clk", "int-clk";
 };
+
+&pruss_soc_bus {
+	status = "okay";
+
+	pruss: pruss@4a300000 {
+		status = "okay";
+
+		pru0: pru@4a334000 {
+			status = "okay";
+		};
+
+		pru1: pru@4a338000 {
+			status = "okay";
+		};
+	};
+};
-- 
1.7.9.5

